account_update.php

```diff
@@ -100,8 +99,8 @@
 # Do not update email for a user verification
 if( !( $t_ldap && config_get( 'use_ldap_email' ) )
 	&& !$t_account_verification ) {
-	if( $f_email != user_get_email( $t_user_id ) ) {
-		$t_update_email = $f_email;
+	if( !is_blank( $f_email ) && $f_email != user_get_email( $t_user_id ) ) {
```

Comment:
This PR is just dealing with the real name and does not change the behavior when changing email address.
Of course, I changed the way it's implemented as I want it the same for all fields.
(false/true instead of null/input value)
So your comments should not block this PR.
You might want to open a new issue to allow users to remove their email address.